ceph/rados/pool_workflows.py

```diff
@@ -0,0 +1,71 @@
+"""
+Module to change pool attributes
+1. Autoscaler tunables
+2. Snapshots
+"""
+import logging
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+
+log = logging.getLogger(__name__)
```

Comment:
let us use the custom log class ahead.

ref : https://github.com/red-hat-storage/cephci/pull/1011

Reply:
Changed to use new log class